react-effects/my-app/src/List.jsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import readItems from './read';
+
+export default function List() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [items, setItems] = useState([]);
+  const [error, setError] = useState();
+
+  // Your code here:
+  //  - When the component mounts:
+  //    - Read the items using `readItems` and update state so the list displays
+  //    - Handle errors from `readItems`
+  useEffect(() => {
+    async function display() {
+      try {
+        const list = await readItems();
+        setIsLoading(false);
+        setItems(list);
+      } catch (error) {
+        setError(error);
```

Comment:
Do you actually see the error message displayed? I think you'll see just the "Loading..." message.

This code is all correct except that you need to set isLoading to false even if there's an error so that you can display the error message.